onnxruntime/core/providers/rocm/tunable/gemm_common.h

```diff
@@ -27,6 +27,9 @@ inline std::string BlasOpToString(BlasOp op) {
       return "N";
     case BlasOp::T:
       return "T";
+    // following is unreachable, compiler is producing false-positive warning, unfortunately.
+    default:
+      return "_";
```

Comment:
> return "_"

Maybe throw an exception if this ever happens, rather than silently return an incorrect value?